site/source/docs/compiling/Deploying-Pages.rst

```diff
@@ -0,0 +1,162 @@
+
+- Be aware of the `Content Security Policy <https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP>`_ rules and make note of what kind of CSP policy the site is planned to run with.
+
+- Be mindful of the `Mixed Content Security <https://developer.mozilla.org/en-US/docs/Web/Security/Mixed_content>`_ restrictions that browsers impose.
+
+- Make sure that the site runs well in private browsing (incognito) mode. For example, this will prevent the site from persisting data to IndexedDB.
+
+- Test that the page works well when put into a background tab. Use the ``blur``, ``focus`` and ``visibilitychange`` DOM events to react to page hide and show events. This is relevant in particular for applications that perform audio playback.
+
+- If the page uses WebGL, make sure that it is able to gracefully handle the WebGL context loss event. Use the `WebGL_lose_context <https://www.khronos.org/registry/webgl/extensions/WEBGL_lose_context/>`_ developer extension to programmatically trigger context loss events when testing.
+
+- Verify that the page works as intended on displays with different ``window.devicePixelRatio`` (DPI) settings, in particular when using WebGL. See `Khronos.org: HandlingHighDPI <https://www.khronos.org/webgl/wiki/HandlingHighDPI>`_. On Windows and OS X, try changing the desktop display scaling setting to test different values of ``window.devicePixelRatio`` that the browser reports.
+
+- Test out that different page zoom levels do not break the site layout, especially when navigating to the page with the browser window already pre-zoomed.
+
+- Likewise, verify that the page layout does not break when resizing the browser window, or when visiting the site having already initially sized the browser window to very small or large size, or to a disproportionate aspect ratio.
+
+- Especially if targeting mobile, be aware of the `<meta viewport> tag <https://developer.mozilla.org/en-US/docs/Mozilla/Mobile/Viewport_meta_tag>`_ for how to develop a site layout that works well on mobile.
+
+- If the page uses WebGL, test out different GPUs on target platforms. In particular, verify the site behavior when simulating the lack of any needed WebGL extensions, and compressed texture format support.
+
+- If using the ``requestAnimationFrame()`` API (i.e. ``emscripten_set_main_loop()`` function) to drive rendering, be aware that the rate at which the function is called is not always 60 Hz, but can vary at runtime e.g. when moving the browser window from one display to another in a multimonitor setup, if the displays have different refresh rates. Update intervals such as 75Hz, 90Hz, 100Hz, 120Hz, 144Hz and 200Hz are becoming more common.
+
+- Simulate lack of any special APIs that the page might need, e.g. Gamepad, Acceleration or Touch Events, and make sure that appropriate error flow is handled in those cases as well.
```

Comment:
The private browsing mode is mentioned a bit above, `"- Make sure that the site runs well in private browsing (incognito) mode. For example, this will prevent the site from persisting data to IndexedDB."`. Good point about slow network connections, I'll make a note about that.